.github/workflows/test.yaml

```diff
@@ -261,6 +261,21 @@ jobs:
         fail_ci_if_error: true
 
 
+  fips-tests:
```

Comment:
you might need to use something like this

we should use tox to manage our CI dependencies 


you should be able to use `pip build` to build the wheel... but you should not worry about this, tox should take care of it
```
  fips-tests:
    # Defined the host in which the container is executed
    runs-on: ubuntu-latest
    # Define container inside which the steps are executed
    container:
      image: photon:5.0
      env:
        NODE_ENV: development
      options: --cpus 2
    # Steps as normal.
    steps:
    - name: Setup the OS
      run: |
        tdnf remove -y toybox
        tdnf update -y
        tdnf install -y coreutils \
                openssl-fips-provider \
                python3-pip \
                python3-pip \
                python3 \
                python3-devel \
                build-essential \
                git \
                shadow \
                tzdata
    - uses: actions/checkout@v3

    - name: Test
      run: |
        tox alldeps-nocov-posix
```